### ISSUE TITLE
Normalize logging behavior of post-write hooks

### ISSUE THREAD

**Issue by kbolko** [MarcusBarnes/mik#189]
Current post-write hooks currently log successes to the mik.log.  This is different than the core MIK tool which only logs failures.  They also don't log to the problem_records.log meaning there is no one stop shop to check for problems.

Both of these issues together make it difficult to detect the failure of a post-write hook.

**Comment by mjordan** [comment]
Logging successful execution of post-write hooks has its uses. I'm currently trying to figure out why none of the three post-write hooks defined in my .ini file are not firing, and it was useful to see that none of them had entries in mik.log.

That said, we can modify the post-write hooks so they log to a file other than mik.log. That way we can can both a cleaner mik.log and confirm that the post-write hook scripts ran as expected. I can make this change today if everyone agrees that it's a satisfactory solution.

**Comment by kbolko** [comment]
Logging success to another file is fine but without splitting the success and failures you're not going to see failures in a stream of thousands of successes.

**Comment by mjordan** [comment]
Success is logged as INFO entries and failure as ERROR entries to make it easy to do a so a quick scan or a grep to split the logs, but if separate logs are more useful we can do that to.

**Comment by mjordan** [comment]
Or another option is to log failure to mik.log and success to a separate log.

**Comment by kbolko** [comment]
I'm suggesting that splitting the two into separate logs is the way to go.  Its much easier to see what the failures are and its more in line with core MIK tool.

**Comment by mjordan** [comment]
OK, I'll do that. @MarcusBarnes any comments?

**Comment by MarcusBarnes** [comment]
I'm OK with splitting warnings and errors into one log and success and info. messages to another, if this makes it easier for people.  Thank you for the input.

**Comment by mjordan** [comment]
I'll have this done by end of the day.

**Comment by mjordan** [comment]
#125 is related to this issue.

**Comment by mjordan** [comment]
OK, I've split out the error and success logs for the validate MODS script and the generate FITS script. File paths are hard coded to the output directory plus descriptive log names. I'd like to make these options as per #125 with decent defaults.

To test, do `git fetch`, then `git checkout issue-189`, then run an MIK job.

**Comment by mjordan** [comment]
Just noticed that if there are any warnings, they are logged to the success log. So that needs a bit more work. But, everything is being logged to the success log, and nothing to mik.log, so at there's that until I can fiddle with Monolog's use of log levels.

**Comment by mjordan** [comment]
Merged into master as of fba21b17817cfc72b3e7247443e64b910fc5386d. Will fix the problem described in the previous comment over the weekend.

**Comment by mjordan** [comment]
OK, both validate_mods.php and generate_fits.php now generate logs named like:

```
postwritehook_generate_fits_success.log
postwritehook_generate_fits_error.log
```

mik.log will contain entries produced by core MIK classes including manipulators.

I've also renamed the log for the object timer post-write hook script to be consistent with the example log filenames provided above. I've merged these changes into master and will pull down them down on the MIK processing VMs, so any job started from now on will use separate logs.
